corehq/apps/saved_reports/tests/test_saved_report_model.py

```diff
@@ -0,0 +1,31 @@
+from django.test import TestCase
+
+from corehq.apps.saved_reports.models import ReportConfig
+from corehq.apps.userreports.reports.view import ConfigurableReportView
+
+
+class SavedReportModelTest(TestCase):
+
+    def test_saved_report_serialized_filters(self):
+        # NOTE:
```

Comment:
:thinking:

Reply:
Thanks for flagging. Wanted to leave a note that the date values in `filters` would be converted to date objects, otherwise this test is pretty lame. Instead of a comment I added a couple of assertions to convey what I wanted.